s3_website.yml

```diff
@@ -0,0 +1,4 @@
+s3_bucket: www.gruntwork.io
+cache_control: max-age=600
+gzip: true
+concurrency_level: 4
```

Comment:
`s3_website` is a very handy tool. We should add it to usage-patterns so customers can use it too!